ios/RNGoogleAds/RNGoogleAdsCommon.m

```diff
@@ -87,55 +87,38 @@ + (GADRequest *)buildAdRequest:(NSDictionary *)adRequestOptions {
 
 + (NSDictionary *)getCodeAndMessageFromAdError:(NSError *)error {
```

Comment:
I notice in Java you changed the method signature but not in Obj-C - I suppose it's best to have them both line up?